Tests/conf.json

```diff
@@ -3184,6 +3189,9 @@
             "integrations": "FeedExpanse",
             "playbookID": "Feed Expanse Test",
             "fromversion": "6.0.0"
+        },
+        {
+            "playbookID": "GreyNoise-test"
```

Comment:
let's remove these lines, partner supported integrations are not tested on our nightly run

Reply:
Sure @guykeller , will remove it